config/production.json

```diff
@@ -17,6 +17,7 @@
 		"code-splitting": true,
 		"community-translator": true,
 		"desktop-promo": true,
+		"google-voucher": true,
```

Comment:
Tested and deployed in `stage` => https://github.com/Automattic/wp-calypso/pull/6057